### PR TITLE
11924 payments unit tests

### PR DESCRIPTION
**Pull request by micahchiang**
## Description
This pull request adds unit tests to the `Payments` component used in `view-payments`

## Testing done
- local, all tests pass

## Screenshots
Screen Shot 2020-09-28 at 11 52 45 AM

## Acceptance criteria
- [x] Tests pass

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/disability-benefits/view-payments/tests/components/view-payments-lists/helpers.js [hunk 1]
```diff
@@ -1,136 +1,91 @@
...
-      bank: 'USAA Federal Savings Bank',
-      account: '1234567890999',
-      rowClass: 'class',
+      returnedCheckAmount: null,
+      returnedCheckCancelDt: '12/01/2012',
+      returnedCheckIssueDt: null,
+      returnedCheckNum: '12365494       ',
+      returnedCheckRo: '17',
+      returnedCheckReason:
+        '6                                                                                                                                                                                                                                                             ',
```
Comment (Mottie) [inline src/applications/disability-benefits/view-payments/tests/components/view-payments-lists/helpers.js:75]: Mistake?

File: src/applications/disability-benefits/view-payments/tests/components/view-payments-lists/helpers.js [hunk 2]
```diff
@@ -1,136 +1,91 @@
...
-      bank: 'USAA Federal Savings Bank',
-      account: '1234567890999',
-      rowClass: 'class',
+      returnedCheckAmount: '$50.00',
+      returnedCheckCancelDt: null,
+      returnedCheckIssueDt: '11/25/2012',
+      returnedCheckNum: '12365494       ',
+      returnedCheckRo: '17',
+      returnedCheckReason:
+        '6                                                                                                                                                                                                                                                             ',
```
Comment (Mottie) [inline src/applications/disability-benefits/view-payments/tests/components/view-payments-lists/helpers.js:86]: here too

**Review by Mottie — CHANGES_REQUESTED** [review]

File: src/applications/disability-benefits/view-payments/tests/components/view-payments-lists/helpers.js (hunk 1 above)
Comment (micahchiang) [reply src/applications/disability-benefits/view-payments/tests/components/view-payments-lists/helpers.js:75]: 👀  yikes

**Review by Mottie — APPROVED** [review]

**Review by cohnjesse — APPROVED** [review]
LGTM

**Pull request merged**